STEERING-COMMITTEE.md

```diff
@@ -1,27 +1,109 @@
+contributors, vendors, and users.
+1. Defining, evolving, and defending a
+[Code of Conduct](CONTRIBUTING.md#code-of-conduct).
+1. Advising the trademark owner on issues relating to the Istio trademark and
+logo.
+1. Setting marketing and advocacy direction for the project; establishing a
+publishing schedule and vetting content, encouraging and assisting in project
+community members, fostering an ecosystem of vendors, creating content for
+conferences, etc.
+1. Controlling and delegating access to, and establishing processes regarding,
+project resources/assets, including but not limited to artifact repositories,
+build and test infrastructure, web sites and their domains, blogs, and social
+media accounts.
+1. Providing neutral mediation as appropriate to try to resolve non-technical
+disputes that arise as part of the project.
+
+## Membership and voting
+1.  The Steering Committee is structured to allow companies who are most
+invested in the success of the Istio project to participate in business and
+non-technical decision-making.
+1.  All members should abide by the project Code of Conduct.
+1.  There are two types of seats on the Steering Committee: Contribution Seats
+and Community Seats.
+1.  Both Contribution and Community Seats will be appointed beginning in
+July 2020. The appointments for the Contribution and Community seat types will
+expire on staggered dates. After the initial term for both seats, all seats
+will have an annual term.
+1.  Contribution Seat terms expire on January 1 and Community Seat terms expire
+on July 1. If necessary, a company holding a Contribution Seat may change the
+appointed individual at any time during the term.
+1.  No Contributing Company can have more than 5 seats in total on the Steering
+Committee.
+    1.  Subsidiaries, an entity in which another company owns more than 50%
+    of the voting or membership interests; and related companies, companies
+    which are controlled by a common third party entity; and affiliate
+    companies, will be treated together as a single entity, referred to as a
+    Contributing Company.
+1.  There shall be nine **Contribution Seats** allocated based on contribution to
+the project. The measurements for effort and contributions can change over time
+via the approval process of the Steering committee members. The current
+measurement is based on by the number of merged pull requests on GitHub over the
+one year period prior to the Contribution Seat assignments:
+    1.  The top three contributing companies to Istio are eligible for
+    Contribution Seats, proportional to their contribution.
+        1.  Each company is allocated one seat;
+        1.  The remaining six seats are allocated based on percentage project
+        contribution, with no Contributing Company exceeding 5 seats in total
+        as outlined in section 6.
+1.  There shall be four **Community Seats.**
+    1.  Two **Community Seats** will be elected by the Istio contributors and
```

Comment:
Can people vote without being maintainers?

Reply:
would be good to describe who can vote, and how many can each person vote and any specific rules with voting.

Reply:
Both are spelled out immediately below.